src/intervals/rounding.jl

```diff
@@ -2,74 +2,176 @@
 # E.g.  +(a, b, RoundDown)
 # Some, like sin(a, RoundDown)  are already defined in CRlibm
 
+# Rounding modes:
+# - :correct  # correct rounding
+# - :fast     # fast rounding by prevfloat and nextfloat  (slightly wider than needed)
+# - :none     # no rounding at all for speed
 
-# import Base: +, -, *, /, sin, sqrt, inv, ^, zero, convert, parse
 
+doc"""Choose rounding mode based on environment variable"""
 
-# unary minus:
--{T<:AbstractFloat}(a::T, ::RoundingMode) = -a  # ignore rounding
+function setup_rounding_mode()
+    if haskey(ENV, "VN_ROUNDING")
+        rounding_mode = Symbol(ENV["VN_ROUNDING"])
```

Comment:
Another question: is this function *useful*? What I mean is that you have to start julia in a certain way (see https://github.com/dpsanders/ValidatedNumerics.jl/pull/207#issue-208675288), so my guess is that you cannot change interactively the rounding mode, or can you?